org.eclipse.buildship.core/src/main/resources/org/eclipse/buildship/core/internal/gradle/java-minimum-supported-toolingAPI.properties

```diff
@@ -0,0 +1,10 @@
+11=4.8
```

Comment:
It seems to me that the data in this file almost matches the [compatibility matrix in the Gradle documentation](https://docs.gradle.org/current/userguide/compatibility.html). It seems like the docs are in fact a bit more restrictive. 

Shouldn't we start with the values in the docs and then adjust the values based on the test failures from the cross-version coverage? Or the numbers in this file is already a result of that?